doc/source/changes/version_0_26.rst.inc

```diff
@@ -70,6 +70,8 @@ Miscellaneous improvements
     ['_name_with_special___char_']
     >>> # special characters \ or / in labels or groups are replaced by an _ when exporting to HDF file
 
+* allowed syntax `axis[other_axis]` (closes :issue:`362`).
```

Comment:
This is a bit short for an explanation. e.g. add something like "to create a group on an axis using all the labels of another axis". Also, an example wouldn't hurt.